dll/win32/devmgr/lang/pt-PT.rc

```diff
@@ -7,10 +7,10 @@ BEGIN
     IDS_NAME "Nome"
     IDS_TYPE "Tipo"
     IDS_MANUFACTURER "Fabricante: %1"
-    IDS_LOCATION "Local: %1"
-    IDS_STATUS "Estado do dispositivo: %1"
+    IDS_LOCATION "Localização: %1"
+    IDS_STATUS "Estado do\ndispositivo: %1"
```

Comment:
Are you sure about using "\n" line break here? Other translations does not seem to do that:

![image](https://user-images.githubusercontent.com/578406/149519155-2de2f225-fb5f-4385-816f-8d73943639fd.png)